packages/gestalt/src/Masonry/defaultTwoColumnModuleLayout.test.js

```diff
@@ -4,7 +4,223 @@ import HeightsStore from './HeightsStore';
 import MeasurementStore from './MeasurementStore';
 import { type Position } from './types';
 
+type Item = {
+  id: string,
+  columnSpan?: number,
+};
+
+const stubCache = (measurements?: { [item: string]: number, ... } = {}) => {
+  let cache = measurements;
+
+  return {
+    get(item: Item) {
+      return cache[item.id];
+    },
+    has(item: Item) {
+      return !!cache[item.id];
+    },
+    set(item: Item, value: number) {
+      cache[item.id] = value;
+    },
+    reset() {
+      cache = {};
+    },
+  };
+};
+
+const positionsStubCache = (positions?: { [item: string]: Position, ... } = {}) => {
+  let cache = positions;
+
+  return {
+    get(item: Item) {
+      return cache[item.id];
+    },
+    has(item: Item) {
+      return !!cache[item.id];
+    },
+    set(item: Item, value: Position) {
+      cache[item.id] = value;
+    },
+    reset() {
+      cache = {};
+    },
+  };
+};
+
 describe('two column layout test cases', () => {
+  test('empty', () => {
```

Comment:
nit: let's call out that these are copied over from the defaultLayout (e.g. `describe('defaultLayout test cases')`).
The ultimate plan is to merge this back into the defaultLayout but these tests are good to ensure we don't break the default layout test cases